Tests/ZMarkupParserTests/HTML/AllTagStyleAttributesHasAddToBuilderDefaultListTests.swift

```diff
@@ -63,4 +63,9 @@ private class StubVisitor: HTMLTagStyleAttributeVisitor {
         count += 1
         return count
     }
+
+    func visit(_ styleAttribute: TextAlignHTMLTagStyleAttribute) -> Int {
+        count += 1
+        return count
+    }
```

Comment:
Test Failed:
ZMarkupParserTests.AllTagStyleAttributesHasAddToBuilderDefaultListTests
testCheckDefaultList, XCTAssertEqual failed: ("8") is not equal to ("7") - Must added new pre-defined

should change `XCTAssertEqual(visitor.count, 7, "Must added new pre-defined HTMLTagStyleAttribute to ZHTMLParserBuilder.styleAttributes")` to
`XCTAssertEqual(visitor.count, 8, "Must added new pre-defined HTMLTagStyleAttribute to ZHTMLParserBuilder.styleAttributes")`

Reply:
Confirm and change! db737b8